.github/workflows/test.yaml

```diff
@@ -21,3 +21,33 @@ jobs:
 
             - name: Lint YAML files
               run: yamllint -c yaml_lint_configuration.yaml .
+    unittests:
+        runs-on: ubuntu-latest
+        steps:
+          - name: Checkout code
+            uses: actions/checkout@v2
+
+          - name: Setup kubectl
+            uses: azure/setup-kubectl@v1
+            with:
+              version: 'v1.20.0'
+
+          - name: Setup Helm
+            uses: azure/setup-helm@v1
+            with:
+              version: 'v3.4.0'
+
+          - uses: actions/setup-python@v4
```

Comment:
Is the Python a dependency for the chart testing/cluster?

Reply:
Yes that is correct.